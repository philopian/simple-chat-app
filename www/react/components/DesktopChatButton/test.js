import React from "react";
import { configure, shallow, mount, render } from "enzyme";
import renderer from "react-test-renderer";
import "raf/polyfill";
import Adapter from "enzyme-adapter-react-16";
import Component from "./index.js";

describe("DesktopChatButton (Snapshot)", () => {
  it("DesktopChatButton renders without crashing", () => {
    const component = renderer.create(<Component />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
