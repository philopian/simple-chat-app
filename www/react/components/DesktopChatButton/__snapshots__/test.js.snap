// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DesktopChatButton (Snapshot) DesktopChatButton renders without crashing 1`] = `
<div
  className="my-component"
>
  <div
    className="sc-bdVaJa hbqgvg"
  >
    <p>
      Chat with someone?
       
      <div
        className="message-icon"
      >
        <i
          aria-hidden="true"
          className="fa fa-commenting"
        />
      </div>
    </p>
  </div>
</div>
`;
